### PR TITLE
OPTM-63 Only show QR scanner button when platform is web, android, iOS.

### PR DESCRIPTION
**Pull request by guisedlythe**
- When QR scanner is not supported for the platform, only allow adding devices with the "device form screen"

OPTM-63 #close

### CODE REVIEW

**Comment by guisedlythe** [comment]
I think I forgot to create a PR for this (minor) change

File: lib/screens/devices/device_list_screen.dart [hunk 1]
```diff
@@ -40,39 +41,29 @@ class _DeviceListScreenState extends State<DeviceListScreen> with LogMixin {
...
   Widget build(BuildContext context) {
-    final consumerWidget = Consumer2<ChangeCallbackNotifier<PeerInfoService>,
-        ChangeCallbackNotifier<PeerService>>(
-      builder: (context, service, peerService, child) => _buildDeviceList(
-        context,
-        service.callbackProvider,
-        peerService.callbackProvider,
-      ),
+    final showQrScannerButton = _showQrScannerButton();
+    final consumerWidget = Consumer2<PeerInfoService, PeerService>(
```
Comment (JacobRoe) [inline lib/screens/devices/device_list_screen.dart]: Shouldn't it be `ChangeCallbackNotifier<PeerInfoService>` and `ChangeCallbackNotifier<PeerService>`?

File: lib/screens/devices/device_list_screen.dart [hunk 2]
```diff
@@ -98,8 +90,11 @@ class _DeviceListScreenState extends State<DeviceListScreen> with LogMixin {
               children: [
                 Spacer(),
                 Text('📪 No devices yet.', style: kHeroFont),
-                Text('Press the button below to scan a QR code.'),
-                Text('Longpress the button below to manually add a device.'),
+                showQrScannerButton
+                    ? Text('Press the button below to scan a QR code.')
+                    : Text('Press the button below to add a device'),
```
Comment (JacobRoe) [inline lib/screens/devices/device_list_screen.dart]: Maybe add a period to the sentence to keep it consistent.

**Review by JacobRoe — CHANGES_REQUESTED** [review]
LGTM except the things I commented on.

**Review by JacobRoe — APPROVED** [review]
Thank you. Looks good.

**Pull request merged**